ts/components/portfolio/OperationsComponent.tsx

```diff
@@ -0,0 +1,134 @@
+import {
+  Body,
+  Grid,
+  Icon,
+  Left,
+  List,
+  ListItem,
+  Right,
+  Row,
+  Text
+} from "native-base";
+import * as React from "react";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+
+import { PortfolioAPI } from "../../api/portfolio/portfolio-api";
+import I18n from "../../i18n";
+import ROUTES from "../../navigation/routes";
+import { CreditCard } from "../../types/portfolio/CreditCard";
+import { Operation } from "../../types/portfolio/types";
+import { PortfolioStyles } from "../styles";
+
+type Props = {
+  parent: string,
+  title: string,
+  totalAmount: string,
+  navigation: NavigationScreenProp<NavigationState>,
+  operations: ReadonlyArray<Operation>
+}
+
+type State = {
+  data: ReadonlyArray<Operation>
+}
+
+/**
+ * Operations" List component
+ */
+export class OperationsList extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props)
+    this.state = { data: Array.from(props.operations) }
+  }
+
+  public render(): React.ReactNode {
+    const { navigate } = this.props.navigation
+    const ops = this.state.data
+
+    if (ops === null || ops === undefined) {
+      return <Text>{"Operations null."}</Text>
+    }
+
+    if (!Array.isArray(ops) || !ops.length) {
+      return <Text>{I18n.t("portfolio.noTransactions")}</Text>
+    }
+
+    return (
+      <Grid>
+        <Row>
+          <Left>
+            <Text style={PortfolioStyles.pfbold}>{this.props.title}</Text>
+          </Left>
+          <Right>
+            <Text>{this.props.totalAmount}</Text>
+          </Right>
+        </Row>
+        <Row>
+          <List
+            removeClippedSubviews={false}
+            dataArray={ops}
+            renderRow={(item): React.ReactElement<any> => (
+              <ListItem
+                onPress={(): boolean =>
```

Comment:
avoid using lambdas in jsx (move the function to the class) - see `jsx-no-lambda` https://github.com/palantir/tslint-react

Reply:
actually this is ok, I've disabled the rule in `master`